package.json

```diff
@@ -91,6 +91,10 @@
     "html-entities": "^2.3.3",
     "i18next": "^22.5.1",
     "immer": "^10.0.2",
+    "js-file-download": "^0.4.12",
+    "mantine-react-table": "^1.3.4",
+    "moment": "^2.29.4",
```

Comment:
We already have dayjs. Do you really need moment?

Reply:
I think it came with mantine-react-table because I don't remember installing it. Is it possible ?